LICENSE

```diff
@@ -19,3 +19,7 @@ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
 LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
+
```

Comment:
Do we all agree on using the MIT license? I know this is the base for the project, but we could look into other licenses that might fit ours better?

Reply:
I'm happy with MIT

Reply:
I really don't know much about licensing, so I'm fine with MIT but if there's a better one I'm open to suggestions.